### DISCUSSION TITLE
Two regex for the same input in Fluent Bit

### DISCUSSION THREAD

**Discussion by BlaBlaGITTE** [fluent/fluent-bit#7516] [Q&A] [answered]
Hi all,
Is it possible to use two different regex for the same log file?

**Comment by patrick-stephens** [comment, marked as answer]
If you mean regex parsers then sure - you can do it in a few ways, e.g. trigger another parser filter on a key or the absence of a key: https://docs.fluentbit.io/manual/pipeline/filters/parser

If a regex does not match then nothing happens either so you can always fire multiple parsers at it.

**Reply by BlaBlaGITTE** [reply]
Solved, thanks :)
